metabase/metabase
# Display official collection tooltips

This PR adds a tooltip to any official collection's "ribbon" icon in Metabase.

### To Verify

1. Spin up Metabase Enterprise and sign in as admin
2. Go through #17125 steps, hover "ribbon" icons, you should see "Official collection" tooltips
3. Go through #17167 steps, hover "ribbon" icons, you should see "Belongs to an Official collection" tooltips

### Demo

<img width="566" alt="CleanShot 2021-08-13 at 22 12 04@2x" src="https://user-images.githubusercontent.com/17258145/129407732-57f490e6-0f80-4c97-a99e-af859330e6c8.png">

Slightly different text in the tooltip for a card from an official collection included in an unofficial dashboard:

![CleanShot 2021-08-13 at 22 13 15@2x](https://user-images.githubusercontent.com/17258145/129407893-7345b183-d252-476c-912e-1a79ad797e69.png)
